pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/PythonPluginIntegrationTest.groovy

```diff
@@ -96,4 +96,43 @@ class PythonPluginIntegrationTest extends Specification {
         result.task(':check').outcome == TaskOutcome.SUCCESS
         result.task(':build').outcome == TaskOutcome.SUCCESS
     }
+
+    def "can build v3 library"() {
+        given:
+        testProjectDir.buildFile << """
+        |plugins {
```

Comment:
Space between the margin `|` and the text is needed.
The indent should stay 4 spaces throughout, even for sample code.